content/overview/quick-start.md

```diff
@@ -3,208 +3,235 @@ layout: overview
+indexing_pipeline.connect("cleaner.documents", "splitter.documents")
+indexing_pipeline.connect("splitter.documents", "embedder.documents")
+indexing_pipeline.connect("embedder.documents", "writer.documents")
+indexing_pipeline.run(data={"sources": ["davinci.txt"]})
+
+text_embedder = OpenAITextEmbedder()
+retriever = ChromaEmbeddingRetriever(document_store)
+template = """Given these documents, answer the question.
+              Documents:
+              {% for doc in documents %}
+                  {{ doc.content }}
+              {% endfor %}
+              Question: {{query}}
+              Answer:"""
+prompt_builder = PromptBuilder(template=template)
+llm = OpenAIGenerator()
 
-# Build a pipeline with a Retriever to get relevant documents to the query and a PromptNode interacting with LLMs using a custom prompt.
-pipeline = build_pipeline(provider, API_KEY, document_store)
+rag_pipeline = Pipeline()
+rag_pipeline.add_component("text_embedder", text_embedder)
+rag_pipeline.add_component("retriever", retriever)
+rag_pipeline.add_component("prompt_builder", prompt_builder)
+rag_pipeline.add_component("llm", llm)
 
-# Ask a question on the data you just added.
-result = pipeline.run(query="Who is the father of Arya Stark?")
+rag_pipeline.connect("text_embedder.embedding", "retriever.query_embedding")
+rag_pipeline.connect("retriever.documents", "prompt_builder.documents")
+rag_pipeline.connect("prompt_builder", "llm")
 
-# For details, like which documents were used to generate the answer, look into the <result> object
-print_answers(result, details="medium")
+query = "How old was he when he died?"
+result = rag_pipeline.run(data={"prompt_builder": {"query":query}, "text_embedder": {"text": query}})
+print(result["llm"]["replies"][0])
 ```
-The output of the pipeline references the documents used to generate the answer:
+{{< /tab  >}}
+
+{{< tab tabName="Pipeline Graph"  >}}
+<div class="row" style="display:flex">
+  <div class="column" style="margin:15px auto" >
+    <p>Indexing Pipeline</p>
+    <img src="/images/indexing.png" width="300" quality="70"/>
+  </div>
+  <div class="column" style="margin:15px auto" >
+    <p>RAG Pipeline</p>
+    <img src="/images/rag.png" width="300" quality="70"/>
+  </div>
+</div>
+{{< /tab  >}}
 
-```text
-'Query: Who is the father of Arya Stark?'
-'Answers:'
-[{'answer': 'The father of Arya Stark is Lord Eddard Stark of '
-                'Winterfell. [Document 1, Document 4, Document 5]'}]
-```
+{{< /tabs >}}
 
-For a hands-on guide on how to build your first RAG Pipeline, see our tutorial.
+For a hands-on guide on how to build your first RAG Pipeline with Haystack 2.0, see our tutorial.
 
-{{< button url="https://haystack.deepset.ai/tutorials/22_pipeline_with_promptnode" text="Tutorial: Creating a RAG Pipeline" color="green">}}
+{{< button url="https://haystack.deepset.ai/tutorials/27_first_rag_pipeline" text="Tutorial: Creating a RAG Pipeline" color="green">}}
```

Comment:
Do we plan to add traceable links to these buttons @TuanaCelik? the other button is for "Docs: Installation"

Reply:
O M G good catch!!!  thank you! And we can, but it's all internal links at this point so not sure how useful they will be. I will add it anyway.